src/System.ServiceModel.Syndication/tests/BasicScenarioTests.cs

```diff
@@ -414,6 +414,94 @@ public static void AtomFeedPositiveTest()
+                    rss20ItemFormatter.ReadFrom(reader);
+                    feedObjct = rss20ItemFormatter.Item;
+                    reader.Close();
+                }
+
+                using (XmlWriter writer = XmlWriter.Create(serializeFilePath))
+                {
+                    Rss20ItemFormatter atomformatter = new Rss20ItemFormatter(feedObjct);
+                    atomformatter.WriteTo(writer);
+                    writer.Close();
+                }
+                // compare file filePath and serializeFilePath
+                XmlDiff diff = new XmlDiff()
+                {
+                    Option = XmlDiffOption.IgnoreComments | XmlDiffOption.IgnorePrefix | XmlDiffOption.IgnoreWhitespace | XmlDiffOption.IgnoreChildOrder | XmlDiffOption.IgnoreAttributeOrder
+                };
+                Assert.True(diff.Compare(filePath, serializeFilePath));
+            }
+            finally
+            {
+                File.Delete(serializeFilePath);
+            }
+        }
+
+        [Fact]
+        [ActiveIssue(24971)]
+        public static void RssFeedPositiveTest()
+        {
+            string dataFile = @"rss_feeds.dat";
+            List<string> fileList = GetTestFilesForFeedTest(dataFile);
+
+            foreach (string file in fileList)
+            {
+                string serializeFilePath = Path.GetTempFileName();
+                try
+                {
+                    SyndicationFeed feedObjct;
+                    using (XmlReader reader = XmlReader.Create(file))
+                    {
+                        Rss20FeedFormatter formatter = new Rss20FeedFormatter();
+                        formatter.ReadFrom(reader);
+                        feedObjct = formatter.Feed;
+                        reader.Close();
+                    }
+
+                    using (XmlWriter writer = XmlWriter.Create(serializeFilePath))
+                    {
+                        Rss20FeedFormatter formatter = new Rss20FeedFormatter(feedObjct);
+                        formatter.WriteTo(writer);
+                        writer.Close();
+                    }
+
+                    CompareHelper ch = new CompareHelper
+                    {
+                        Diff = new XmlDiff()
+                        {
+                            Option = XmlDiffOption.IgnoreComments | XmlDiffOption.IgnorePrefix | XmlDiffOption.IgnoreWhitespace | XmlDiffOption.IgnoreChildOrder | XmlDiffOption.IgnoreAttributeOrder
+                        },
+                        AllowableDifferences = GetRssFeedPositiveTestAllowableDifferences()
+                    };
+                    Assert.True(ch.Compare(file, serializeFilePath), $"Failed File Name:{file}");
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine();
+                    Console.WriteLine("------------------------------");
+                    Console.WriteLine($"Failed File Name:{file}");
```

Comment:
Put the message in the exception instead of using Console.WriteLine.

Reply:
Done.